심화요구사항.md

```
@@ -1,29 +1,16 @@
 # 심화 요구사항
 
-## Bean 다루기
+- [ ]  다음의 정적 리소스를 서빙하여 프론트엔드와 통합해보세요. API 스펙을 준수했다면 잘 동작할거예요.
 
-- [x]  Repository 구현체 중에 어떤 구현체를 Bean으로 등록할지 Java 코드의 변경 없이 application.yaml 설정 값을 통해 제어해보세요.
+> [fe_1.0.0.zip]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)